src/handlebar-helpers.js

```diff
@@ -91,6 +91,26 @@ const helpers = {
       if (value === '') return false;
       return true;
     });
+  },
+  startswith: function(str, ...args) {
+    if (typeof str !== 'string') return false;
+    return args.some((arg) => {
```

Comment:
i believe string.startsWith expects a string as input. should probably check if arg is a string as well

Reply:
No, this will work just fine. the function will do appropriate casting

```javascript
'1'.startsWith(1) // true
```

Reply:
The tests cover this

```javascript
assert.ok(startswith('hello world', null, 1, 'test', 'hel'), 'string match found');
assert.notOk(endswith('hello world', null, 1, 'test','bye'), 'string match not found');
```